datalad/utils.py

```diff
@@ -2531,6 +2531,39 @@ def split_cmdline(s):
     return args
 
 
+def download_url(url, dest, overwrite=False):
```

Comment:
if anywhere, better to be in `support.network` I guess?

Reply:
FTR: I am interpreting this comment as "should be moved" and not as "should not exist", although both interpretations match the statement.

Reply:
Moved.